File: src/main/java/com/example/interviewlandbackend/model/Subscribe.java
package com.example.interviewlandbackend.model;

import javax.persistence.*;
import java.io.Serializable;


@Entity
@Table(name = "subscribes_user")
public class Subscribe implements Serializable {


    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    private String email;


    public Subscribe() {
    }

    public Subscribe( String email) {
        this.email = email;
    }


    public int getId() {
        return id;
    }

    public String getEmail() {
        return email;
    }

}

File: src/main/java/com/example/interviewlandbackend/model/Subscribe.kt
package com.example.interviewlandbackend.model

import java.io.Serializable
import java.util.*
import javax.persistence.*


@Entity
@Table(name = "subscribes_user")
data class Subscribe @JvmOverloads constructor(

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Int?=0,

    val email: String,

    @ManyToOne
    @JoinColumn(name = "contentId")
    val content: Content,

    val date: Date?= Date(System.currentTimeMillis())


):Serializable{


}
